packages/client/src/components/MovieOverview/MovieOverview.jsx

```diff
@@ -0,0 +1,48 @@
+import { useTheme } from '@mui/material/styles';
+import { MovieListProvider } from '../../context/movieListContext';
+import { Box, Container } from '@mui/material';
+import { Categories } from '../Categories/Categories';
+import { SearchBar } from '../SearchBar/SearchBar';
+import { MovieList } from './MovieList';
+import React from 'react';
+
+export const MovieOverview = ({ isFavouritePage }) => {
+  const theme = useTheme();
+
+  return (
```

Comment:
Would be nice to have that title here 🙏🏽

Reply:
We will update it as part of the next PR. Thank you for the review.